documentation/src/main/asciidoc/topics/rest_examples/get_v2_tasks_list.adoc

```diff
@@ -0,0 +1 @@
+GET /rest/v2/tasks?action=list
```

Comment:
`action=list` isn't required. (if I understood the code correctly)

Reply:
Fixed, thanks